src/components/Clock/Clock.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { ClockShape } from '../shapes/ClockShape';
+
+export class Clock extends React.Component {
+  state = {
+    time: new Date(),
+  }
+
+  componentDidMount() {
+    setInterval(() => {
+      const date = new Date();
+
+      // eslint-disable-next-line
+      console.log(date.toLocaleTimeString());
+      this.setState({ time: date });
+    }, 1000);
+  }
```

Comment:
you also need to clear the timer when the Clock is removed from the DOM, cause now the clock is still ticking in the console even after the Clock component is destroyed
hint: use componentWillMount and clearInterval
https://reactjs.org/docs/state-and-lifecycle.html